lib/src/strings.dart

```diff
@@ -72,6 +72,15 @@ String _yamlEncodeDoubleQuoted(String string) {
 /// It is important that we ensure that [string] is free of unprintable
 /// characters by calling [assertValidScalar] before invoking this function.
```

Comment:
I think this comment is wrong - assertValidScalar does not check for unprintable characters.
```suggestion
/// Precondition `!_hasUnprintableCharacters(string)`.
```

Reply:
Good point!